composer.json

```diff
@@ -1,29 +1,29 @@
 {
-    "name": "joomla-projects/joomla-browser",
-    "description": "joomla-browser Codeception Module",
-    "keywords":["BDD", "acceptance testing", "tdd", "joomla"],
-    "homepage":"https://docs.joomla.org/Testing_Joomla_Extensions_with_Codeception",
-    "license": "GPL-2.0+",
-    "authors": [
+    "name"          : "joomla-projects/joomla-browser",
+    "description"   : "joomla-browser Codeception Module",
+    "keywords"      : ["BDD", "acceptance testing", "tdd", "joomla"],
+    "homepage"      : "https://docs.joomla.org/Testing_Joomla_Extensions_with_Codeception",
+    "license"       : "GPL-2.0+",
+    "authors"       : [
         {
-            "name": "Puneet Kala",
-          "email": "puneet.kala@community.joomla.org"
+          "name"    : "Puneet Kala",
+          "email"   : "puneet.kala@community.joomla.org"
         },
         {
-          "name": "Javier Gomez",
-          "email": "javier.gomez@community.joomla.org"
+          "name"    : "Javier Gomez",
+          "email"   : "javier.gomez@community.joomla.org"
         }
     ],
     "require": {
-        "php": ">=5.3.0"
+        "php"                                   : ">=5.5.9",
```

Comment:
According to the new requirements in the latest `consolidation/robo` source and some Symfony dependencies, I see no way but to upgrade the requirement of Joomla Browser (Robo-based) to a minimum of PHP 5.5.9.

That means we will limit the environments a Robo script can run in to 5.5.9 and above.

However, if we really need to add PHP 5.3 and 5.4 for testing purposes, we could later do workarounds in the actual testing scripts (not the Docker-based environment since that one should run in latest PHP 7.x) so they can run in such PHP versions.

Leaving it for discussion @yvesh @javigomez @puneet0191